### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by hjain-git**

### CODE REVIEW

File: run_conf/bstack-local-parallel.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "auth": {
+    "username": "BROWSERSTACK_USERNAME",
+    "access_key": "BROWSERSTACK_ACCESS_KEY"
+  },
+  "browsers": [
+    {
+      "browser": "chrome",
+      "os": "Windows 10",
+      "versions": ["88"]
```
Comment (princebaretto99) [inline run_conf/bstack-local-parallel.json]: Lets update this with "latest" to be futureproof

File: run_conf/bstack-local-parallel.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    {
+      "browser": "firefox",
+      "os": "OS X Big Sur",
+      "versions": ["86"]
```
Comment (princebaretto99) [inline run_conf/bstack-local-parallel.json]: Lets update this with "latest" to be futureproof

File: run_conf/bstack-local-parallel.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    {
+      "browser": "edge",
+      "os": "Windows 10",
+      "versions": ["88"]
```
Comment (princebaretto99) [inline run_conf/bstack-local-parallel.json]: Lets update this with "latest" to be futureproof

File: run_conf/bstack-local-parallel.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    {
+      "browser": "chrome",
+      "os": "OS X Catalina",
+      "versions": ["89"]
```
Comment (princebaretto99) [inline run_conf/bstack-local-parallel.json]: Lets update this with "latest" to be futureproof

File: run_conf/bstack-local-single.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
+{
+  "auth": {
+    "username": "BROWSERSTACK_USERNAME",
+    "access_key": "BROWSERSTACK_ACCESS_KEY"
+  },
+  "browsers": [
+    {
+      "browser": "chrome",
+      "os": "Windows 10",
+      "versions": ["88"]
```
Comment (princebaretto99) [inline run_conf/bstack-local-single.json]: Lets update this with "latest" to be futureproof

File: run_conf/bstack-parallel.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
+{
+  "auth": {
+    "username": "BROWSERSTACK_USERNAME",
+    "access_key": "BROWSERSTACK_ACCESS_KEY"
+  },
+  "browsers": [
+    {
+      "browser": "chrome",
+      "os": "Windows 10",
+      "versions": ["88"]
```
Comment (princebaretto99) [inline run_conf/bstack-parallel.json]: Similarly update to "latest"

File: run_conf/bstack-parallel.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    {
+      "browser": "firefox",
+      "os": "OS X Big Sur",
+      "versions": ["86"]
```
Comment (princebaretto99) [inline run_conf/bstack-parallel.json]: Similarly update to "latest"

File: run_conf/bstack-parallel.json [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    {
+      "browser": "edge",
+      "os": "Windows 10",
+      "versions": ["88"]
```
Comment (princebaretto99) [inline run_conf/bstack-parallel.json]: Similarly update to "latest"

File: run_conf/bstack-parallel.json [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    {
+      "browser": "chrome",
+      "os": "OS X Catalina",
+      "versions": ["89"]
```
Comment (princebaretto99) [inline run_conf/bstack-parallel.json]: Lets also add 2-3 more versions with latest-1

File: run_conf/bstack-single.json [hunk 10]
```diff
@@ -0,0 +1,35 @@
+{
+  "auth": {
+    "username": "BROWSERSTACK_USERNAME",
+    "access_key": "BROWSERSTACK_ACCESS_KEY"
+  },
+  "browsers": [
+    {
+      "browser": "chrome",
+      "os": "Windows 10",
+      "versions": ["88"]
```
Comment (princebaretto99) [inline run_conf/bstack-single.json]: Update to latest

**Review by princebaretto99 — CHANGES_REQUESTED** [review]
Please delete the .DS_Store files as well.

File: run_conf/bstack-local-parallel.json (hunk 1 above)
Comment (hjain-git) [reply run_conf/bstack-local-parallel.json]: done

File: run_conf/bstack-local-parallel.json (hunk 3 above)
Comment (hjain-git) [reply run_conf/bstack-local-parallel.json]: done

File: run_conf/bstack-local-single.json (hunk 5 above)
Comment (hjain-git) [reply run_conf/bstack-local-single.json]: done

File: run_conf/bstack-parallel.json (hunk 6 above)
Comment (hjain-git) [reply run_conf/bstack-parallel.json]: done

File: run_conf/bstack-parallel.json (hunk 7 above)
Comment (hjain-git) [reply run_conf/bstack-parallel.json]: done

File: run_conf/bstack-parallel.json (hunk 8 above)
Comment (hjain-git) [reply run_conf/bstack-parallel.json]: done

File: run_conf/bstack-parallel.json (hunk 9 above)
Comment (hjain-git) [reply run_conf/bstack-parallel.json]: done

File: run_conf/bstack-single.json (hunk 10 above)
Comment (hjain-git) [reply run_conf/bstack-single.json]: done

File: run_conf/bstack-local-parallel.json (hunk 2 above)
Comment (hjain-git) [reply run_conf/bstack-local-parallel.json]: done

File: run_conf/bstack-local-parallel.json (hunk 4 above)
Comment (hjain-git) [reply run_conf/bstack-local-parallel.json]: done

**Review by princebaretto99 — APPROVED** [review]

**Pull request merged**